start-mycroft.sh

```diff
@@ -190,7 +190,9 @@ if [[ "${1}" == "restart" ]] || [[ "${_opt}" == "restart" ]] ; then
 fi
 _params=$@
 
-check-dependencies
+if ! grep -i cli <<< "${_params}" ; then 
```

Comment:
Hey I just tried to run this, but it seems like all the parameters have been shifted by the time it gets to this check.

Can we swap it to only skip the check for "cli"?
```
if [[ ! "${_opt}" == "cli" ]] ; then 
```